MinecraftClient/Resources/Translations/Translations.Designer.cs

```diff
@@ -2426,7 +2426,15 @@ internal class Translations {
                 return ResourceManager.GetString("cmd.inventory.shiftclick", resourceCulture);
             }
         }
-        
+
+        internal static string cmd_inventory_shiftrightclick //shiftrightclick translations link
```

Comment:
You need to use Visual Studio to add a new translation string instead of directly modifying source. I'll do that for you.

Reply:
well i didn't know about how this one work so can i leave it to you?